docs/advisories/cve_2017_5715.md

```diff
@@ -0,0 +1,78 @@
+## CVE-2017-5715
+
+| NAME         	| CVE-2017-5715                                                                                                                                                                                          	|
+|--------------	|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|
+| Description  	| Systems with microprocessors utilizing speculative execution and branch prediction may allow unauthorized disclosure of information to an attacker with local user access via a side-channel analysis. 	|
+| Source       	| https://nvd.nist.gov/vuln/detail/CVE-2017-5715                                                                                                              |
+| NVD Severity 	| medium (attack range: local) |
+| Last Updated  | Jan 01 2018 |
+
+##  Solutions
+
+This hardware exploit requires the installation of the Linux Kernel >= 4.4.110.
```

Comment:
Let's reorder:

* All platforms are affected, not just AWS
* kops can run an image of your choice
* Linux kernel versions needed: 4.4: >= 4.4.110, ...
* By default, kops runs an image that includes the 4.4 kernel.  An updated image is available with 4.4.110
* If running another image please update to a fixed image, which must be provided by your distro
* After update you will have to do an update & rolling-update.

Reply:
Or maybe we swap `kops can run` with `linux kernel versions needed`... :-)